README.md

```diff
@@ -61,9 +61,9 @@ Flags:
 
 ## Code Base Starting Point
 
- Using the LSP/Protocal from ACME https://github.com/fhs/acme-lsp and stripping out anything related to serving, proxy or anything. Just keeping the types for communication
+Using the LSP/Protocal from Golang https://github.com/golang/tools/tree/master/gopls/internal/lsp/protocol and stripping out anything related to serving, proxy or anything. Just keeping the types for communication
```

Comment:
Is something specific stopping us from being able to use https://pkg.go.dev/go.lsp.dev/protocol ?

Reply:
As far as I'm aware, I don't think that you can import stuff from a Go library that's in a directory called 'internal'. Unfortunately, the types are in such a directory. If there's another way to get around this, I'd be really interested!

Reply:
Am I missing something? that pkg was not an `internal` library.